actions/class.Runner.php

```diff
@@ -71,6 +77,13 @@ class taoQtiTest_actions_Runner extends tao_actions_ServiceModule
 {
     use RunnerToolStates;
 
+    /**
+     * Controls whether launching a new delivery suspends other sessions by the same user.
+     *
+     * @var string
+     */
+    public const FEATURE_FLAG_PAUSE_CONCURRENT_SESSIONS = 'FEATURE_FLAG_PAUSE_CONCURRENT_SESSIONS';
```

Comment:
Lets use it on README.md only. The constant is the value itself and we do not use for featureFlags.

```suggestion
```

Reply:
Hmm.. the `FeatureFlagCheckerInterface` itself uses this approach to expose some feature flags publicly, so I used the same approach here. ltiDeliveryProvider is also using this technique to expose `FEATURE_FLAG_MAINTAIN_RESTARTED_DELIVERY_EXECUTION_STATE` ([here](https://github.com/oat-sa/extension-tao-ltideliveryprovider/blob/master/controller/DeliveryTool.php)).

I don't have a strong preference in this case, and the constant is currently just used once, so I'm removing this const and adding it to the README instead.

Reply:
Yes, indeed about `FeatureFlagCheckerInterface` it uses, but the problem is that it is not sustainable as it is on tao-core and we should document it on README.md instead. As you can see, not all FeatureFlags are there and they vary from extension to extension. More visible as a config in README.md. 

Thanks!

Reply:
Added to the README: 1ed567a3ea9055015ee2f1b57076def513eb7bb9